src/libraries/Microsoft.Extensions.Logging.Abstractions/src/Microsoft.Extensions.Logging.Abstractions.csproj

```
@@ -17,6 +17,7 @@ Microsoft.Extensions.Logging.Logger&lt;T&gt;
 Microsoft.Extensions.Logging.LoggerMessage
 Microsoft.Extensions.Logging.Abstractions.NullLogger</PackageDescription>
     <DisableSourceGeneratorPropertyName>DisableMicrosoftExtensionsLoggingSourceGenerator</DisableSourceGeneratorPropertyName>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Needs to specify the version too, as described here: https://github.com/dotnet/runtime/blob/release/7.0/docs/project/library-servicing.md

```suggestion
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <ServicingVersion>1</ServicingVersion>
```